ci-operator/config/red-hat-data-services/openvino_model_server/red-hat-data-services-openvino_model_server-v2022.3-release.yaml

```diff
@@ -0,0 +1,216 @@
+      bin: # What is the "bin tag"?
+        paths: # This lets us copy stuff out of the image right? Does it happen at the beginning or end of the build? I'm assuming the end
+        - destination_dir: lib/redhat # Is this relative to context_dir or no?
+          source_path: /custom_nodes/lib/libcustom_node_add_one.so
+    build_args: # Basing this on https://github.com/openshift/ci-tools/blob/91f8a8f09c57e637223ca312cdc454a08a305fd7/pkg/api/types.go#L1968 right?
+      - name: NODE_NAME
+        value: add_one
+      - name: NODE_TYPE
+        value: cpp
+    to: "openvino/model_server-custom-node:add_one" # This seems to be the output image name, we can use this in future layers right?
+  - dockerfile_path: src/custom_nodes/Dockerfile.redhat
+    context_dir: src/custom_nodes
+    inputs:
+      bin:
+        paths:
+        - destination_dir: lib/redhat
+          source_path: /custom_nodes/lib/libcustom_node_east_ocr.so
+    build_args:
+      - name: NODE_NAME
+        value: east_ocr
+      - name: NODE_TYPE
+        value: cpp
+    to: "openvino/model_server-custom-node:east_ocr"
+  - dockerfile_path: src/custom_nodes/Dockerfile.redhat
+    context_dir: src/custom_nodes
+    inputs:
+      bin:
+        paths:
+        - destination_dir: lib/redhat
+          source_path: /custom_nodes/lib/libcustom_node_face_blur.so
+    build_args:
+      - name: NODE_NAME
+        value: face_blur
+      - name: NODE_TYPE
+        value: cpp
+    to: "openvino/model_server-custom-node:face_blur"
+  - dockerfile_path: src/custom_nodes/Dockerfile.redhat
+    context_dir: src/custom_nodes
+    inputs:
+      bin:
+        paths:
+        - destination_dir: lib/redhat
+          source_path: /custom_nodes/lib/libcustom_node_horizontal_ocr.so
+    build_args:
+      - name: NODE_NAME
+        value: horizontal_ocr
+      - name: NODE_TYPE
+        value: cpp
+    to: "openvino/model_server-custom-node:horizontal_ocr"
+  - dockerfile_path: src/custom_nodes/Dockerfile.redhat
+    context_dir: src/custom_nodes
+    inputs:
+      bin:
+        paths:
+        - destination_dir: lib/redhat
+          source_path: /custom_nodes/lib/libcustom_node_image_transformation.so
+    build_args:
+      - name: NODE_NAME
+        value: image_transformation
+      - name: NODE_TYPE
+        value: cpp
+    to: "openvino/model_server-custom-node:image_transformation"
+  - dockerfile_path: src/custom_nodes/Dockerfile.redhat
+    context_dir: src/custom_nodes
+    inputs:
+      bin:
+        paths:
+        - destination_dir: lib/redhat
+          source_path: /custom_nodes/lib/libcustom_node_model_zoo_intel_object_detection.so
+    build_args:
+      - name: NODE_NAME
+        value: model_zoo_intel_object_detection
+      - name: NODE_TYPE
+        value: cpp
+    to: "openvino/model_server-custom-node:model_zoo_intel_object_detection"
+  # Also, all that gets built here is a dependency for the following, but the above could be built in parallel
```

Comment:
All image builds run in parallel

Reply:
Is there a way to enforce a barrier or something? Some earlier images are needed by later stages